worker/lib/index.ts

```diff
@@ -280,6 +291,119 @@ async function setup(): Promise<express.Application> {
 		},
 	);
 
+	// give SSH keys to access DUT (is this the best way of doing this???)
+	app.post(
+		'/ssh/setup',
+		jsonParser,
+		async (
+			req: express.Request,
+			res: express.Response,
+			next: express.NextFunction,
+		) => {
+			try {
+				const keyPath = join(homedir(), 'id');
+				const keyPathPub = join(homedir(), 'id.pub');;
+				// need to write these strings to files
+				writeFileSync(keyPath, req.body.id);
+				writeFileSync(keyPath, req.body.id_pub);
+				await execSync('ssh-add -D');
+				await execSync(`ssh-add ${keyPath}`);
+				res.send('OK');
+			} catch (err) {
+				console.error(err);
+				next(err);
+			}
+		},
+	);
+
+	app.post(
+		'/dut/exec',
```

Comment:
This is to exec a command on the DUT? Can we get some comments?

Reply:
Correct, this is to execute a command in the DUT host OS - will add comments